be/src/vec/aggregate_functions/aggregate_function.h

```diff
@@ -34,10 +34,16 @@
 
 namespace doris::vectorized {
 
+const static std::string DECIMAL256_FUNCTION_SUFFIX {"_decimal256"};
```

Comment:
This static var should be moved to a cpp file and it should have global visibility.
Just leave `extern std::string DECIMAL256_FUNCTION_SUFFIX` in header file.